src/Microsoft.Framework.PackageManager/Utils/LockFileUtils.cs

```diff
@@ -63,9 +65,7 @@ public static LockFileTargetLibrary CreateLockFileTargetLibrary(IPackage package
             // it has the correct casing that runtime needs during dependency resolution.
             lockFileLib.Name = correctedPackageName ?? package.Id;
             lockFileLib.Version = package.Version;
-
-            var files = package.GetFiles().Select(p => p.Path.Replace(Path.DirectorySeparatorChar, '/')).ToList();
-
+            var files = library.Files.Select(p => p.Replace(Path.DirectorySeparatorChar, '/'));
```

Comment:
I wasn't suggesting using `Path.Combine`. It was more along the lines of - if we store these files with transformed paths to start with, `Path.Combine` would continue working without issues and we'll avoid iterating over this list twice.